### PR TITLE
Update packaging

### PR DESCRIPTION
**Pull request by fmarier**
Fixes to make the package build with the latest client.

Remaining issues: `ConfArgParse` and `python2-pythondialog`

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -3,7 +3,7 @@ Section: web
 Priority: optional
 Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
 Uploaders: Francois Marier <francois@debian.org>
-Build-Depends: debhelper (>= 9), python-all, dh-python
+Build-Depends: debhelper (>= 9), python-all, dh-python, python-m2crypto, python-zope.component, python-augeas, python-psutil, python-jsonschema, python-requests, python-nose, python-openssl, python-crypto, python-nosexcover, python-tox, python-mock, python-confargparse
```
Comment (kuba) [inline debian/control]: Most of these are necessary for runtime. Is `Build-Depends` sufficient? Shouldn't it go to `Depends` instead?

File: debian/docs [hunk 2]
```diff
@@ -1,2 +1,2 @@
 EULA
```
Comment (kuba) [inline debian/docs]: `EULA` is now a symlink, so it might be necessary to fix this line as well

**Comment by fmarier** [comment]
@kuba I've addressed the two problems you found.

**Comment by kuba** [comment]
LGTM, thanks :+1:

**Comment by fmarier** [comment]
BTW, I don't have merge powers in this repo, so someone else needs to do it :)

**Comment by jdkasten** [comment]
Thanks!

**Pull request merged**